archinstall/lib/general.py

```diff
@@ -185,6 +185,8 @@ def __init__(self,
 		self.callbacks = callbacks
 		self.peak_output = peak_output
 		self.environment_vars = environment_vars
+		# define the standard locale for command outputs. For now the C ascii one"
+		self.environment_vars['LC_ALL'] = "C"
```

Comment:
I'm thinking this might be bad if the users use `SysCommand(Worker)` to execute things in their profiles, and it reverts their global locale. I think we'll have to internally set `LC_ALL=C` like you've done in `luks.py` to avoid this.

Reply:
No. The default i set in general.py (LC_ALL=C), only affects commands executed INSIDE Archinstall, so we have a guaranteed stable environment. It does not touch user's own environment.
To make matters easier for a developer, i now allow to change the locale on a command by command base (as i did in Luks ). This is a change i just uploaded, my previous version